opensfm/src/sfm/observation.h

```diff
@@ -15,4 +15,16 @@ struct Observation {
   double scale{1.};
   Eigen::Vector3i color;
   int id{0};
+
+  Observation(double x, double y, double s, uint8_t r, uint8_t g, uint8_t b, size_t id,
+            float _angle, float _resp, float _size, int _class_id)
+  : point(x, y), scale(s), color(r, g, b), id(id),
+    angle(_angle), response(_resp), size(_size), class_id(_class_id)
+    { }
+  
+  float angle;
+  float response;
+  // int octave; //this is probably the scale
```

Comment:
You can remove it, we don't use it (and you're right, it is the inverse scale ;))